GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/LooseObjectStepTests.cs

```diff
@@ -0,0 +1,139 @@
+﻿using GVFS.FunctionalTests.FileSystemRunners;
+using GVFS.FunctionalTests.Tools;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.RegularExpressions;
+
+namespace GVFS.FunctionalTests.Tests.EnlistmentPerFixture
+{
+    // TODO(Mac): Before these tests can be enabled PostFetchJobShouldComplete needs

```

Comment:
Is this comment (and the `MacTODO.M4` category) needed?

Reply:
Good point, these comments are copy/paste issues and I've updated so we should run on Mac